src/index.html

```diff
@@ -22,6 +22,413 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="Company name logo Moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
```

Comment:
The `is-active` class suggests that there is some styling for the active state of the link. Ensure that there is corresponding CSS that differentiates the active link from others.